src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+};
+
+type TodoCondition = 'neutral' | 'seving' | 'editing';
```

Comment:
Consider renaming `TodoCondition` to `TodoStatus` for better readability and consistency with the rest of the code. 🤔

Reply:
I decided on `TodoCondition ` , because, every todo has own status completed or not.
So it can be confusing if:
`type TodoStatus = 'neutral' | 'seving' | 'editing';`
&
`todo.completed = true | false`   (it looks more like status)

or not ?